interactive/__init__.py

```diff
@@ -1,41 +1,97 @@
 from common.consts import PROJECT_PATH
 
-
 GRID_NROW_NUMBER = 3
 EXAMPLE_PATH = PROJECT_PATH / "interactive" / "examples"
 CATEGORY_DESCR = {
```

Comment:
I would suggest to move this big dict into separate file (could be consts etc.) and import it here instead of declaring

Reply:
Done